### PR TITLE
[Mailer] Redlink bridge

### PR DESCRIPTION
**Pull request by plotkabytes**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | yes
| License       | MIT
| Doc PR        | symfony/symfony-docs#18339

Added [Redlink](https://docs.redlink.pl) mailer bridge.

### CODE REVIEW

File: src/Symfony/Component/Mailer/Bridge/Redlink/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+6.3
```
Comment (OskarStark) [inline src/Symfony/Component/Mailer/Bridge/Redlink/CHANGELOG.md]: 6.4

Comment (plotkabytes) [reply src/Symfony/Component/Mailer/Bridge/Redlink/CHANGELOG.md]: Fixed

File: src/Symfony/Component/Mailer/Bridge/Redlink/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+6.4
+-----
+
+ * Added the bridge
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Mailer/Bridge/Redlink/CHANGELOG.md:7]:
```suggestion
 * Add the bridge
```

File: src/Symfony/Component/Mailer/Bridge/Redlink/Transport/RedlinkApiTransport.php [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+        HttpClientInterface $client = null,
+        EventDispatcherInterface $dispatcher = null,
+        LoggerInterface $logger = null
+    ) {
+        parent::__construct($client, $dispatcher, $logger);
+    }
+
+    public function __toString(): string
+    {
+        return sprintf('redlink+api://%s', $this->getEndpoint());
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Mailer/Bridge/Redlink/Transport/RedlinkApiTransport.php:50]: why `+api` and not only the non suffixed variant?

Comment (nicolas-grekas) [reply src/Symfony/Component/Mailer/Bridge/Redlink/Transport/RedlinkApiTransport.php:50]: ah, ok, this is the mailer, not the notifier :)

**Comment by fabpot** [comment]
@plotkabytes Are you still interested in finishing this PR? Do you need any help?

**Comment by fabpot** [comment]
Closing as there is no feedback. Feel free to reopen when you have time. Thank you.

**Pull request closed without merging**